### PR TITLE
[ANCHOR-704] Document user_action_required_by

### PR DESCRIPTION
**Pull request by JiahuiWho**
1. Clean up deprecated `amount_fee` and update platform transaction schema
2. Add user_action_required_by to related RPC methods
    NotifyInteractiveFlowCompletedRequest
    NotifyTransactionOnHoldRequest
    NotifyOffchainFundsPendingRequest 
    NotifyOffchainFundsAvailableRequest
    RequestCustomerInfoUpdateRequest
    RequestOffchainFundsRequest
    RequestOnchainFundsRequest
    RequestTrustRequest
4. Add use instruction in SEP-6 and SEP-24
    platforms/anchor-platform/admin-guide/sep6/configuration.mdx
    platforms/anchor-platform/admin-guide/sep24/configuration.mdx

### CODE REVIEW

**Comment by lijamie98** [comment]
There are 8 actions modified. But the PR has 148 files modified. Is that expected?

**Comment by philipliu** [comment]
I think `yarn api` regenerated the API docs for SDP and Horizon. Can we remove those from the PR to make it easier to review?

**Comment by JiahuiWho** [comment]
There will me more files than just 8 because cleaning up `amount_fee ` from all existing rpc methos

File: platforms/anchor-platform/admin-guide/sep24/configuration.mdx [hunk 1]
```diff
@@ -109,6 +114,8 @@ SECRET_SEP24_MORE_INFO_URL_JWT_SECRET="your encryption key shared with your busi
 
 `SECRET_SEP24_INTERACTIVE_URL_JWT_SECRET` and `SECRET_SEP24_MORE_INFO_URL_JWT_SECRET` are encryption keys that the Anchor Platform will use to generate short-lived tokens it will add to the URLs provided to the wallet. Your business server must also have these keys in its environment so it can verify the token's signature.
 
+`SEP24_INITIAL_USER_DEADLINE_SECONDS` is the time in seconds that defines the user action deadline before a transaction automatically moves to the next status. It determines the `user_action_required_by` field, which indicates the deadline for statuses with automatic movement. Check [JSON-RPC Methods][json-rpc-methods] for usage examples.
```
Comment (philipliu) [inline platforms/anchor-platform/admin-guide/sep24/configuration.mdx:117]:
```suggestion

`SEP24_INITIAL_USER_DEADLINE_SECONDS` is the time in seconds a user has to act before the transaction automatically moves to the next status. It determines the `user_action_required_by` field, which indicates the deadline for statuses with automatic movement. Check [JSON-RPC Methods][json-rpc-methods] for usage examples.
```

File: platforms/anchor-platform/admin-guide/sep6/configuration.mdx [hunk 2]
```diff
@@ -164,6 +164,10 @@ SECRET_CALLBACK_API_AUTH_SECRET="a secret used to sign JWTs"
 
 The above tells the Anchor Platform to include a [JWT][how-to-use-jwt], signed with the configured secret, in the `Authorization` header of requests made to `/callbacks/<callback endpoint>` so your server can authenticate the Anchor Platform before processing requests.
 
+See the [KYC API][platform-api-kyc] and [Rates API][sep38] for details on the endpoints that must be implemented on your business server.
```
Comment (philipliu) [inline platforms/anchor-platform/admin-guide/sep6/configuration.mdx:167]: Should we be linking to the callback docs instead of `sep38`?

**Review by philipliu — APPROVED** [review]
LGTM. Let's get one more review though.

**Comment by Ifropc** [comment]
We should not merge this PR before https://github.com/stellar/stellar-docs/pull/872 
There's going to be a lot of merge conflicts as the result, and I think it's better to resolve conflicts here than there (it's already a huge change in Elliot's PR)

**Comment by JiahuiWho** [comment]
Closing this one to create a new PR to avoid mass merge

**Pull request closed without merging**